### PR TITLE
[12.0] IMP l10n_it_fatturapa_in adding "Recompute XML fields" button when fields like "invoices_number" and "registered" are not valorized

### PR DESCRIPTION
**Pull request by eLBati**
This may happen in case of temporary errors while importing from SDI

--
Confermo di aver firmato il CLA https://odoo-community.org/page/cla e di aver letto le linee guida su https://odoo-community.org/page/contributing

### CODE REVIEW

**Review by mrcast — APPROVED** [review]
Functional test OK!

File: l10n_it_fatturapa_in/models/attachment.py [hunk 1]
```diff
@@ -73,6 +73,12 @@ def onchagne_datas_fname(self):
     def get_xml_string(self):
         return self.ir_attachment_id.get_xml_string()
 
+    @api.multi
+    def recompute_xml_fields(self):
+        for att in self:
+            att._compute_xml_data()
+            att._compute_registered()
```
Comment (sergiocorato) [inline l10n_it_fatturapa_in/models/attachment.py:80]: Perchè non?
```suggestion
        self._compute_xml_data()
        self._compute_registered()
```

Comment (eLBati) [reply l10n_it_fatturapa_in/models/attachment.py]: Ci sta. Modificato. Grazie

**Review by sergiocorato — APPROVED** [review]
Grazie

**Pull request merged**